apps/src/code-studio/pd/workshop_dashboard/reports/local_summer_workshop_daily_survey/results.jsx

```diff
@@ -114,7 +115,7 @@ export default class Results extends React.Component {
                     {
                       this.state.facilitatorIds.map((id, j) => (
                         <td key={j}>
-                          {this.props.thisWorkshop[session]['facilitator'][question_key][id]}
+                          {he.decode(this.props.thisWorkshop[session]['facilitator'][question_key][id])}
```

Comment:
Is this related to stripping out blank results?

Reply:
Nope - this is for https://trello.com/c/69kfFZLW/634-malformatting-of-apostrophes-in-survey-responses